domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -616,6 +591,10 @@ class TopicListControllerTest {
     return Date().time - NINE_DAYS_IN_MS
   }
 
+  private fun retrieveTopicSummaryFromTopicList(index: Int): TopicSummary {
+    return topicListController.getTopicList().value!!.getOrThrow().getTopicSummary(index)
```

Comment:
Hmm maybe wait until #2253 is merged, first, since it will largely change your approach here.

Reply:
Done